controllers/mysql_container.go

```diff
@@ -161,7 +161,14 @@ func (r *MySQLClusterReconciler) makeV1AgentContainer(cluster *mocov1beta2.MySQL
 }
 
 func (r *MySQLClusterReconciler) makeV1SlowQueryLogContainer(sts *appsv1ac.StatefulSetApplyConfiguration, force bool) *corev1ac.ContainerApplyConfiguration {
-	if !force {
+	isNotNil := func(sts *appsv1ac.StatefulSetApplyConfiguration) bool {
```

Comment:
Why do you define a function for this?
This would be simpler.
```go
stsINotNil := !(sts == nil || sts.Spec == nil || sts.Spec.Template == nil || sts.Spec.Template.Spec == nil)
```

Reply:
Thanks!
https://github.com/cybozu-go/moco/pull/373/commits/0ecaac56591ac5b339ab72c023fab4099de5bb75